hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -216,6 +228,14 @@ public void init(
                 migrateFrom(deserializedVersion);
             }
         }
+        runPostInitTasks();
+    }
+
+    private void runPostInitTasks() {
+        for (Runnable task : postInitTasks) {
+            task.run();
+        }
+        postInitTasks.clear();
```

Comment:
Maybe a dumb question, but would this be better named `postMigrateTasks`? 🤔

Reply:
yea... I should name this as postMigrateTasks. I have to apply this fix in two places :D